### PR TITLE
Adds simple paths for getting realistic test data.

### PR DESCRIPTION
**Pull request by strangeties**
1) Updates forsys_request_params to be more modular.
2) Introduces url parameters for making it easier to run forsys patchmax on test data sets.

Specifically, among other things, realistic test data sets will ideally ...
1) cover a few planning areas that are ~100k acres throughout the Sierra Nevada region (4.5k stands for 300m x 300m granularity)
2) focus on 5 contrasting conditions: california_spotted_owl, storage, functional_fire, forest_structure, max_sdi

Example calls:
- For for Middle Fork test area: http://localhost:8000/planscape-backend/forsys/generate_project_areas/single_scenario/?request_type=5
- For any combination of HUC-12 areas: http://localhost:8000/planscape-backend/forsys/generate_project_areas/single_scenario/?request_type=2&huc12_names=Little+Silver+Creek-Silver+Creek&huc12_names=Slab+Creek

Example planning area:
image

TODO, but not in this pr: suggest Patchmax parameters for additional realistic parameters and constraints:
1) projects should cover 30% of planning area
2) slope must be below 30 degrees, areas must be less than 200 ft from roads
3) individual project areas should be at least 100 acres (~5 stands for 300m x 300m granularity)

### CODE REVIEW

File: src/planscape/forsys/forsys_request_params.py [hunk 1]
```diff
@@ -350,3 +304,152 @@ def _get_default_planning_area(self) -> MultiPolygon:
...
+    else:
+        raise Exception("ranking request type was not recognized")
+
+
+# Returns ForsysGenerationRequestParams based on url parameter value for the
+# parameter name in _URL_REQUEST_PARAMS_TYPE.
+def get_generation_request_params(
+        request: HttpRequest) -> ForsysGenerationRequestParams:
+    params = request.GET
+    type = ForsysGenerationRequestParamsType(int(
```
Comment (elsiel23) [inline src/planscape/forsys/forsys_request_params.py:440]: these types a little bit hard to interpret without looking at the classes. i would suggest something like this for consistency:

DATABASE
ALL_DEFAULTS (we know that use of defaults implies URL) 
HUC12s_WITH_DEFAULTS
SILVER_CREEK_HUC12s_WITH_DEFAULTS
etc.

**Review by elsiel23 — APPROVED** [review]

**Comment by elsiel23** [comment]
one more thought i have around readability of code. can we move the polygon definitions in forsys_request_params.py into a separate file that way we can have more of them and they are easier to edit/identify?

File: src/planscape/forsys/forsys_request_params.py (hunk 1 above)
Comment (strangeties) [reply src/planscape/forsys/forsys_request_params.py:440]: Done.

**Comment by strangeties** [comment]
sure! Done - I've moved get_default_project_areas and get_default_planning_area to a separate file.

And thanks for reviewing.

**Pull request merged**